include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -90,12 +94,19 @@ struct Base {
     return p;
   }
 
+  T** put() noexcept {
+    //ASSERT(p_ == nullptr);
```

Comment:
was this temporary? seems like we should have the assert uncommented.

Reply:
great catch, i got it back in.  i wasn't sure which flavor to use (upper case ASSERT is a windows thing.    i went with lowercase assert which is CRT thing and matches the rest of ORT.

---
In reply to: [354034969](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354034969) [](ancestors = 354034969)

Reply:
i added it to your whiteboard

---
In reply to: [354620668](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354620668) [](ancestors = 354620668,354034969)